src/zimmount/zimmount.h

```diff
@@ -0,0 +1,38 @@
+#ifndef OPENZIM_ZIMMOUNT_H
+#define OPENZIM_ZIMMOUNT_H
+
+#include <string>
+#include <unordered_map>
+
+#define ZIM_PRIVATE
```

Comment:
I don't think we need `ZIM_PRIVATE` here.
Using the public API of libzim should be enough.